controllers/storage.go

```diff
@@ -22,13 +22,10 @@ import (
 	dspav1alpha1 "github.com/opendatahub-io/data-science-pipelines-operator/api/v1alpha1"
 )
 
-const storageSecret = "minio/secret.yaml.tmpl"
-
 var storageTemplates = []string{
 	"minio/deployment.yaml.tmpl",
 	"minio/pvc.yaml.tmpl",
 	"minio/service.yaml.tmpl",
-	storageSecret,
```

Comment:
This will break deployments that are using minio (so no minio specified), as it will not deploy the randomly generated secret. Even though minio is unsupported, we should consider keeping this around to for simplifying minio deployments.